portfolio/src/main/webapp/index.html

```diff
@@ -61,6 +61,12 @@ <h1 id="random-facts">Random Facts</h1>
 <p>Whenever someone asks for a fun fact, my go to response is
    how I've broken both of my arms.
    Once when I was on a skateboard and once when I was on a bike.
-   One thing that others point out is how I can remember the tiniest things.
+   One thing that others point out is how I can remember the tiniest details about things.
+<form action="/data" method="POST">
```

Comment:
`method="POST"`

Use lowercase for attribute values.

See https://google.github.io/styleguide/htmlcssguide.html#Capitalization

```html
<form action="/data" method="post">
```

Reply:
Done.